src/main/kotlin/io/github/mojira/arisa/modules/PrivacyModule.kt

```diff
@@ -14,7 +14,7 @@ class PrivacyModule(
     private val patterns: List<Regex> = listOf(
         """.*\(Session ID is token:.*""".toRegex(),
         """.*--accessToken ey.*""".toRegex(),
-        """.*\b[A-Z0-9]{17}\b.*""".toRegex(),
+        """.*\b(?=[^\s]*[A-Z])(?=[^\s]*[0-9])[A-Z0-9]{17}\b.*""".toRegex(),
```

Comment:
This mismatch between `\b` before and behind the regex but `\s` in the look aheads now causes a different issue where any substring consisting only of letters or only of digits matches when followed by `\b` chars which are not in `\s`. E.g. the following will now be matched:
```
AAAAAAAAAAAAAAAAA;;;;;;;;;;;;;;;;;;;;;;;;;;;;;1
```
To solve this either use the exact regex (with leading and trailing `.*`) as described in your comment https://github.com/mojira/arisa-kt/issues/571#issuecomment-788057416, or use the alternative with `\b` as described in https://github.com/mojira/arisa-kt/issues/571#issuecomment-788070724.